fetch.bs

```diff
@@ -2071,15 +2071,16 @@ for each associated <a for="fetch group">fetch record</a> whose
 <a for=connection>connection pool</a> consists of zero or more
 <dfn lt=connection export id=concept-connection>connections</dfn>. Each
 <a>connection</a> is identified by an <b>origin</b> (an
-<a for=/>origin</a>) and <b>credentials</b> (a boolean).
+<a for=/>origin</a>), <b>credentials</b> (a boolean), and
+<b>networkPartitionKey</b> (a <a for=/>site</a> list or null).
```

Comment:
I suggest we call this key and lead with it. So key, origin, credentials. And per below its type can be network partition key.

Reply:
Done. Renamed it in HTTP cache partitions section as well, but stuck with networkPartitionKey in the logic to find a connection, since that's a longer method.